✨ Add URL scheme support

SenpaiHunters opened the pull request:
- Implemented URL command handling in AppDelegate
- Registered custom URL scheme 'loop' in Info.plist
- Added URLCommandHandler to process incoming URLs
- Configured Apple Event handling for URL interactions

SenpaiHunters committed:
✨ Add URL scheme support for Loop app

- Implemented URL command handling in AppDelegate
- Registered custom URL scheme 'loop' in Info.plist
- Added URLCommandHandler to process incoming URLs
- Configured Apple Event handling for URL interactions

mrkai77 commented on Loop/Utilities/URLCommandHandler.swift:
```diff
@@ -0,0 +1,412 @@
+        print("[URLHandler] Found \(visibleWindows.count) eligible windows")
+
+        // Find target window
+        guard let window = findTargetWindow(from: visibleWindows) else {
```
I think it makes more sense to find the window with `WindowEngine.getTargetWindow()`?

SenpaiHunters committed:
Merge branch 'develop' into url-scheme

SenpaiHunters committed:
Merge branch 'develop' into url-scheme

SenpaiHunters committed:
✨ Enhance URL command handling

- Added new action, keybind, and list command formats to the URLCommandHandler documentation.
- Improved command processing to handle new action and keybind commands.
- Implemented output buffering for list commands to enhance user experience.
- Updated error handling and logging for better debugging and user feedback.
- Refactored command handling methods for clarity and maintainability.

MrKai77 committed:
⚡ Remove redundant `WindowEngine.getTargetWindow()`s

As that is already included in `findTargetWindow()`

MrKai77 reviewed (approved):
Seems good, will release a new development build with these changes!
